### PR TITLE
update: add support for postcss 8 & update dependencies

### PR DESCRIPTION
**Pull request by okikio**
- support postcss 8 
- fix bugs in tests (I found some inconsistencies); 
- update dependencies; 
- update rollup build process

**postcss-import@latest doesn't seem to import without .scss at the end**

### CODE REVIEW

**Comment by sinankeskin** [comment]
Any news on this PR?

Latest version autoprefixer throws an error because of the postcss 7 dependency of this package.

Thank you.

**Comment by okikio** [comment]
@jonathantneal can this be merged or are more test required?

**Comment by okikio** [comment]
**Note:** the postcss-import issue stems from  postcss-import@latest not importing files without .scss at the end of the file name.

E.g. 
```scss
// This is required at the end of an import for postcss-import to work properly
@import "components.scss";

```

To fix this problem in my PR I changed the order of the postcss plug-ins and had postcss-scss run first, and then postcss-import.

**Comment by majo44** [comment]
Please release !

File: package.json [hunk 1]
```diff
@@ -25,24 +25,24 @@
   },
   "dependencies": {
     "@csstools/sass-import-resolve": "^1.0.0",
-    "postcss": "^7.0.14",
-    "sass": "^1.16.1",
+    "postcss": "^8.2.4",
```
Comment (majo44) [inline package.json]: Move to peerDependencies - https://evilmartians.com/chronicles/postcss-8-plugin-migration

Comment (okikio) [reply package.json]: I have moved postcss to the peerDependency.

**Comment by sinankeskin** [comment]
Hi.

I can confirm that all issues have been resolved with this PR. Here is my fork: https://github.com/sinankeskin/postcss-sass

You can release confidently.

Thank you.

**Comment by amprew** [comment]
Is there any update on releasing this PR? I guess the only thing left is to bump the version? Thanks!

Think it has to be a repository owner

**Comment by sinankeskin** [comment]
> Is there any update on releasing this PR? I guess the only thing left is to bump the version? Thanks!
> 
> Think it has to be a repository owner

I guess he is not available right now.

While waiting to release you use my fork with `npm i https://github.com/sinankeskin/postcss-sass.git`

**Comment by amprew** [comment]
Thanks, @sinankeskin. Didn't realise it was distributable like that

**Comment by miguelcobain** [comment]
@jonathantneal any news on the release? It would be great if we could use sass with PostCSS 8. :)

**Pull request merged**